docs/nodes/nodes-library/nodes/GoogleSheets/README.md

```diff
@@ -34,39 +34,137 @@ The final workflow should look like the following image.
-6. Click on ***Execute Node*** to run the node.
-
-![Using the Set node to set data to be inserted by the Google Sheets node](./Set_node.png)
+The example workflow uses the Set node to generate data that we want to add to the Google Sheet. We can use other nodes, like [HTTP Request node](../../core-nodes/HTTPRequest/README.md), to get the data that we want to add to the Google Sheet.
+
+1. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+2. Enter `Name` in the ***Name*** field.
+3. Enter the name of the house in the ***Value*** field.
+4. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+5. Enter `Rent` in the ***Name*** field.
+6. Enter the rent of the house with the currency (example $1000) in the ***Value*** field.
+7. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+8. Enter `City` in the ***Name*** field.
+9. Enter the name of the city in the ***Value*** field.
+10. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
+11. Enter `id` in the ***Name*** field.
+12. Click on the gears icon next to the ***Value*** field click on ***Add Expression***.
+13. Paste the following expression `{{Math.floor(Math.random()*1000)}}`. This expression will generate a three-digit random number.
+14. Toggle ***Keep Only Set*** to true.
+15. Click on ***Execute Node*** to run the node.
+:::
 
+![Using the Set node to set data to be inserted in the Google Sheets node](./Set_node.png)
 
+  
 ### 3. Google Sheets node (Append)
 
+This node will append the data in the Google Sheet.
+
 1. First of all, you'll have to enter credentials for the Google Sheets node. You can find out how to do that [here](../../../credentials/Google/README.md), in the section 'Google Drive / Sheets API'.
 2. Select 'OAuth2' in the ***Authentication*** field.
 3. Select 'Append' from the ***Operation*** dropdown list.
 4. Copy the string of characters located between `/d/` and `/edit` in your spreadsheet URL. Paste that string in the ***Sheet ID*** field.
-5. In the ***Range*** field, enter the range of columns to append the data to in your spreadsheet. Make sure that your range includes enough columns for all the data included in the Set node.
-6. Enter `id` in the A1 cell and `name` in the B1 cell  of the spreadsheet.
-7. Click on ***Execute Node*** to run the workflow.
+5. In the ***Range*** field, enter the range of columns to append the data to, in your spreadsheet. Make sure that your range includes enough columns for all the data in the Set node. For this workflow, enter `A:D` in the ***Range*** field.
+6. Click on the ***Add Option*** button and select 'Value Input Mode' from the dropdown list.
+7. Select 'User Entered' from the ***Value Input Mode*** dropdown list.
+8. In your spreadsheet, enter `ID`, `Name`, `Rent`, and `City` in the cells A1, B1, C1, and D1 respectively.
+9. Click on ***Execute Node*** to run the workflow.
 
```

Comment:
Can you please add an explanation about what the readers are expected to see in the screenshot below?